Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/PaywallViewMode+Extensions.swift

```diff
@@ -8,11 +8,43 @@
 import Foundation
 import RevenueCat
 
-internal extension PaywallViewMode {
+enum PaywallTesterViewMode: String {
+    case fullScreen = "Full Screen"
+    case sheet = "Sheet"
+    case footer = "Footer"
+    case condensedFooter = "Condensed Footer"
+}
```

Comment:
More a question to get context than a comment. Are those rawValue coming from somewhere? Like, are those used for decoding / encoding? Because I see the name uses the same strings.

Reply:
Ahhh, I was going to use them as a display name in the UI but ended up making with a `name` variable in the extension instead 😊  Going to remove these!